packages/app/src/domain/vaccine/vaccine-delivery-and-administrations-area-chart.tsx

```diff
@@ -9,8 +11,23 @@ import { VaccineDeliveryAndAdministrationsTooltip } from './components/vaccine-d
 import {
   DeliveryAndAdministrationData,
   VaccineDeliveryAndAdministrationsValue,
-} from './data-selection/selected-delivery-and-administration-data';
-import { useVaccineNames } from './use-vaccine-names';
+} from './data-selection/select-delivery-and-administration-data';
+
+const vaccines = [
+  'pfizer',
+  'moderna',
+  'astra_zeneca',
+  'cure_vac',
+  'janssen',
+  'sanofi',
+] as (keyof Pick<
+  VaccineDeliveryAndAdministrationsValue,
+  'pfizer' | 'moderna' | 'astra_zeneca' | 'cure_vac' | 'janssen' | 'sanofi'
+>)[];
```

Comment:
Why is this typed so complex compared to `as const`?
```suggestion
const vaccines = [
  'pfizer',
  'moderna',
  'astra_zeneca',
  'cure_vac',
  'janssen',
  'sanofi',
] as const;
```

Reply:
Because it's been used as an indexer, so simply using strings will make the compiler sad

Reply:
It looks like the compiler currently complains: https://github.com/minvws/nl-covid19-data-dashboard/pull/2855/checks?check_run_id=2617693033

I also think the compiler is complaining because the `vaccines` array contains more types of vaccines compared to the vaccines coming from lokalize. On my local checkout the following results in no TS errors:
```tsx
const vaccines = [
  'astra_zeneca',
  'janssen',
  'moderna',
  'pfizer',
  // 'cure_vac',
  // 'sanofi',
] as const;
```

Reply:
See line 42, 111, 114 and 115 for reference

Reply:
If remove those vaccine names then we'll have to make a manual change here once those vaccines are on the market. We added the entire list so that BE only has to add the values and this chart will automatically 'grow' accordingly...

Reply:
Sure, makes sense, but another way of fixing is by adding `cure_vac` and `sanofi` in lokalize. Otherwise BE is still unable to add values without our help.

Reply:
They are in the Lokalise, no?

Reply:
At least, in my nl_export.json they are...

Reply:
that's odd, because they are not part of the dataset:
![image](https://user-images.githubusercontent.com/73584448/118777860-d1833780-b889-11eb-96b2-b503ec052510.png)

Reply:
Oh, I guess I have an old export locally then. These keys have probably been removed during cleanup, I suppose.

Reply:
whoopsy, must have been my bad then

Reply:
don't forget to re-add those vaccines again with the cli :-)